bodhi/server/models.py

```diff
@@ -735,6 +735,75 @@ def unpush(self, koji):
 
 
 class Update(Base):
+    """
+    This model represents a package.
+
+    This model uses single-table inheritance to allow for different package types.
+
+    Attributes:
+        title (unicode): The update's title which uniquely identifies the update.
+        autokarma (bool): A boolean that indicates whether or not the update will
+            be automatically pushed when the stable_karma threshold is reached.
+        stable_karma (int): A positive integer that indicates the amount of "good"
+            karma the update must receive before being marked as stable.
+        unstable_karma (int): A positive integer that indicates the amount of "bad"
+            karma the update must receive before being marked as unstable.
+        requirements (unicode): A list of taskotron tests that must pass for this
+            update to be considered stable. Note: This does not appear to actually
+            work and is likely to be replaced by PolicyEngine in the long-term.
```

Comment:
I'm not sure I'd put that note here just yet because I'm not sure if I used it correctly when I tried.